src/functions-reference/matrix_operations.Rmd

```diff
@@ -2120,6 +2120,9 @@ The determinant of A
 The log of the absolute value of the determinant of A
 `r since("2.0")`
 
+`real` **`log_determinant_spd`**`(matrix A)`<br>\newline
+The log of the absolute value of the determinant of the symmetric, positive-definite matrix A.
```

Comment:
It'd be nice overall to use `` `A` ``, but given we're super inconsistent about that in the doc, I'm going to approve as is.